doc/orchestrator.rst

```diff
@@ -165,3 +165,287 @@ A full example of configuring KeyDB to run in SmartSim is shown below
   export REDIS_CONF=/path/to/keydb.conf
 
   # run smartsim workload
+
+Multidb
+=======
+
+To support computationally intensive tasks like scientific simulations,
+SmartSim provides functionality to automate the deployment of multiple
+databases on an HPC cluster. As data volume increases, consider augmenting
+the number of sharded databases or provision multiple standalone databases,
+to ensure the workload can scale to meet resource demands. To ensure that data
+retrieval, updates, and synchronization are carried out smoothly and efficiently
+across multiple database nodes, unique identifier names are given to each database
+during initialization. Below is a general example of launching multiple databases
+with a unique identifier and distributing data between the launched databases.
+
+Defining workflow stages requires the utilization of functions associated
+with the Experiment object. The Experiment object is intended to be instantiated
+once and utilized throughout the workflow runtime. Begin by creating an Experiment
+object and assign it to the 'exp' variable. In this example, we instantiate the
+Experiment object with the name 'getting-started-multidb' and the 'launcher=slurm'
+configuration.
+
+.. code-block:: python
```

Comment:
Always leave a blank line between `.. code-block:: python` and the code itself. For some reason, the doc processor requires it and will fail rendering the code in some cases otherwise.